src/common/wordLists/compiled/commonWords.json

```diff
@@ -2606,7 +2601,6 @@
   "SEABED",
   "SEALED",
   "SEAMAN",
-  "SEAMEN",
```

Comment:
un-related to the offensiveness, this was just deemed to not be "common"

Reply:
> un-related to the offensiveness

😅*Giving my chuckling inner 5yo a stern introspective look.*